lib/vfscore/stdio.c

```diff
@@ -205,11 +207,12 @@ static struct vfscore_file  stdio_file = {
 	 * operation. However it is not properly handled in the
 	 * current implementation. */
 	.f_count = 2,
+	.f_ep = UK_LIST_HEAD_INIT(stdio_file.f_ep)
 };
 
 void init_stdio(void)
 {
-	int fd;
+	int fd __maybe_unused;
```

Comment:
If this can be done separately, you can take it out and I can approve it today even 😄